hibernate-core/src/main/java/org/hibernate/metamodel/mapping/internal/BasicValuedCollectionPart.java

```diff
@@ -238,9 +239,8 @@ public void visitJdbcTypes(
 		action.accept( getJdbcMapping() );
 	}
 
-	//
```

Comment:
I'm curious... why remove this?

Reply:
the commented code? I really do not remember